server/models/Order.js

```diff
@@ -292,7 +293,7 @@ function defineModel() {
     // Register user as collective backer
     const member = await this.collective.findOrAddUserWithRole(
       { id: this.CreatedByUserId, CollectiveId: this.FromCollectiveId },
-      roles.BACKER,
+      this.tier?.type === tiers.TICKET ? roles.ATTENDEE : roles.BACKER,
```

Comment:
I'm not sure `tier` will always be loaded when calling this helper, like here for example: https://github.com/opencollective/opencollective-api/blob/488eec825a7cd78a59b8c25708c078ed73c42a96/server/paymentProviders/paypal/webhook.ts#L127

Reply:
@Betree : I've changed the code to obtain the tier information based on the `TierID` instead rather than relying on the `tier` object. I think this makes more sense. Let me know. 🤔